data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg

```diff
@@ -835,7 +835,7 @@
 
         [message]
             speaker="Ro'Arthian"
-            message= _ "Very well, Tallin. We accept under the terms that your people will forever leave us in peace after this."
+            message= _ "Very well, Tallin. We accept under the terms that after this your people will forever leave us in peace ."
```

Comment:
I don't think there's much of a difference so the original is fine.

Reply:
You can wait for other feedback first.

Reply:
Sure go for it.

Reply:
Either works for me, but if you go ahead with this change please remove the space from before the full-stop.

Reply:
I made it a full stop.